_posts/2018-01-02-bbc-newshack.html

```diff
@@ -0,0 +1,46 @@
+<p>Navigating data (through audio): How can we envisage audio charts? This task felt too titanesque to undertake in a couple of days.</p>
+<p>As we went on, it became obvious that most suggestions were revolving aroung people's interactions; with various ideas around fact-checking, offering context, feedback and comments. We settled on exploring a few of those ideas, acting together in a voice assistant enable user conversations via our Comments section.</p>
+<img src="/assets/uploads/2018/01/newshack_brainstorm.jpg" alt="Ideas from the brainstorming session." />
+
+<h2>FT Conversation</h2>
+<h3>What it is</h3>
+<p>On FT.com, user comments add value to a piece. With FT conversation, we wanted to drive user engagement for our readers who already own voice devices. Comments are often free-flowing and we wondered whether it was possible to increase the quality of those comments if they were recorded through a VUI, with sentiment analysis and a "swear jar" built in.</p>
+<p>Voice interfaces offer a reflexion time while the comment is being read back for approval; the vocal component makes it easier for commenters to see there are humans at the other end, and could foster a rich conversation.</p>
+<p>We envisage the app being used during periods of leisure, at home, maybe after a day at work. FT readers would launch the application and be reminded of the articles they've read during the day, and prompted to listen to existing comments or leave a new one. Before someone can post their comment, they must answer a question relating to the article correctly.</p>
+<img src="/assets/uploads/2018/01/userjourney_twine.png" alt="FUser Journey mapped with Twine" />
+
+<h3>How it works</h3>
+<p>We prototyped FT Conversation to be a Google Home application, as we were getting close to launching <a href="https://labs.ft.com/games/2017/09/18/make-connections.html">Make Connections</a> at the time and were already familiar with the platform. We also used <a href="https://twinery.org" target="_blank">Twine</a> to map out all the interactions within the app. We have found the tool useful in the past to help identify dead-ends and greay areas.</p>
+<p>In our demo scenario, the user gets told which article they've read last and if they'd like to leave a comment. If they choose to do so, they get presented with a question relating to the article. Due to hackathon settings, only a few questions were written for selected stories. If they answer correctly, they can record a comment which will be translated to text and posted on their behalf. Before the comment is posted, it is analysed through an API we built on the day, using the Natural Language Toolkit (NLTK) that returns the overall sentiment and how many swear words the message might contain. It gives the user time to ponder whether they want to post their comment or change it.</p>
```

Comment:
which article they last read

Reply:
"only a few questions were written for selected stories." (although from our learnings while building Make Connection, we can easily auto-generate simple questions relevant to an article)